### PR TITLE
Annotate 4-byte alignment from spec

### PR DESCRIPTION
**Pull request by fabioarnold**

### CODE REVIEW

**Comment by fabioarnold** [comment]
I could fix most tests by using `readFileAllocOptions` and pass an alignment.
But I think we only need 4-byte alignment when loading glb or any of the binaries. When loading any gltf json files, we don't need strict alignment. However, the `parseGlb` function is currently non-public.

@kooparse What do you prefer? A shared `parse` function taking aligned data or having two separate functions for gltf and glb files?

File: src/main.zig [hunk 1]
```diff
@@ -75,7 +75,7 @@ pub const Data = struct {
 arena: *ArenaAllocator,
 data: Data,
 
-glb_binary: ?[]const u8 = null,
+glb_binary: ?[]const align(4) u8 = null,
```
Comment (fabioarnold) [inline src/main.zig:78]:
```suggestion
glb_binary: ?[]align(4) const u8 = null,
```

**Comment by kooparse** [comment]
Thanks for the PR, I'm gonna check that before the end of this week :)

**Review by kooparse — APPROVED** [review]
I added a commit to fix tests (I choose only one `parse` function and fixing errors by providing alignment to `readFileAlloc`). 

LGTM ; thanks a lot! :)

**Pull request merged**